core/App/types/credential-status.ts

```diff
@@ -0,0 +1,3 @@
+export enum CredentialStatus {
+  REVOKED = 0,
```

Comment:
`Revoked`?

Reply:
In the branding guidelines there could be other statuses than `revoked` but those are just conceptual from a UI standpoint. For now I can keep it as a `boolean` prop